From 2ea39d893245ed5bcfa3c636e6d87add4e8425a3 Mon Sep 17 00:00:00 2001
From: Tomas Babej <tbabej@redhat.com>
Date: Wed, 24 Apr 2013 14:12:50 +0200
Subject: [PATCH 98/98] Make gecos field editable in Web UI

This patch exposes user entry gecos field in Web UI.

https://fedorahosted.org/freeipa/ticket/3569
---
 install/ui/user.js | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/install/ui/user.js b/install/ui/user.js
index d0f1f5af00ee3ab228d0af646d0eed256f1d6bff..7ebb867a975d3ab71626c6130d4731c88b16dab8 100644
--- a/install/ui/user.js
+++ b/install/ui/user.js
@@ -90,7 +90,8 @@ IPA.user.entity = function(spec) {
                         'sn',
                         'cn',
                         'displayname',
-                        'initials'
+                        'initials',
+                        'gecos'
                     ]
                 },
                 {
@@ -664,4 +665,4 @@ IPA.user.reset_password_acl_evaluator = function(spec) {
     return that;
 };
 
-IPA.register('user', IPA.user.entity);
\ No newline at end of file
+IPA.register('user', IPA.user.entity);
-- 
1.8.3.1

